db/functions.sql

```diff
@@ -21,11 +21,6 @@ SET check_function_bodies = false;
 SET client_min_messages = warning;
 SET escape_string_warning = off;
 
--- [Required] These functions require the plpgsql language --
--- If this line fails, make sure plpgsql isn't already created in the target db
--- (9.0 and later versions of Postgres support CREATE OR REPLACE LANGUAGE)
-CREATE LANGUAGE plpgsql;
```

Comment:
Do we have a minimum version of Postgres specified anywhere?  This was clearly required before 9.0, so we should ensure that 9.0 is a specified minimum...